### PR TITLE
Remove autogenerated docs

### PR DESCRIPTION
**Pull request by lognaturel**
It's not clear what the intended audience of these docs was. Most users should focus on the README/tutorials to learn about usage and extension points. Most of what was included in the jsdoc is not intended to be a public API to develop against.

If someone really needs this, they can let us know why.

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -9,18 +9,11 @@
...
         "sourceType": "module",
         "ecmaVersion": 2020
     },
-    "settings": {
-        "jsdoc": {
-            "tagNamePreference": {
-                "returns": "return"
-            }
-        }
-    },
```
Comment (eyelidlessness) [inline .eslintrc.json:23]: I'm *perfectly fine* with removing the `settings` portion, and if it turns out we're not depending on the plugin removing that too. But to be clear, this is not part of the doc generation.

File: tutorials/README.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# Enketo Core tutorials
+
+* [Getting started](getting-started.md)
+* [Configuration](configuration.md)
+* [Development](development.md)
+* [Events](events.md)
+* [Creating/extending widgets](widgets.md)
```
Comment (eyelidlessness) [inline tutorials/README.md:7]: Why not just let GitHub provide the navigation?

File: .eslintrc.json (hunk 1 above)
Comment (lognaturel) [reply .eslintrc.json:23]: Yeah, I guess we might as well keep that consistent. Will undo.

File: tutorials/README.md (hunk 2 above)
Comment (lognaturel) [reply tutorials/README.md:7]: I... don't know?

**Review by eyelidlessness — APPROVED** [review]

**Pull request merged**